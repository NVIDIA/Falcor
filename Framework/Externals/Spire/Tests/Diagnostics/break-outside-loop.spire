// `break` where it isn't allowed

void foo() { break; }
