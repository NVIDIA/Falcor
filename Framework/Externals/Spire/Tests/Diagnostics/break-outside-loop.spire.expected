result code = -1
standard error = {
Tests/Diagnostics/break-outside-loop.spire(3): error 30003: 'break' must appear inside loop constructs.
}
standard output = {
}
